.github/workflows/upstream_testing.yml

```diff
@@ -10,4 +10,4 @@ jobs:
     uses: "nautobot/nautobot/.github/workflows/plugin_upstream_testing_base.yml@develop"
     with:  # Below could potentially be collapsed into a single argument if a concrete relationship between both is enforced
       invoke_context_name: "NAUTOBOT_FLOOR_PLAN"
-      plugin_name: "nautobot-plugin-floor-plan"
+      plugin_name: "nautobot-app-floor-plan"
```

Comment:
Future work to change upstream to use `app_name` here?

Reply:
Yes